lib/zclient.c

```diff
@@ -392,7 +392,6 @@ enum zclient_send_status zclient_send_hello(struct zclient *zclient)
 		stream_putc(s, zclient->redist_default);
 		stream_putw(s, zclient->instance);
 		stream_putl(s, zclient->session_id);
-		stream_putc(s, 0); /* receive_notify - removed */
```

Comment:
I would combine this commit with the previous one.

Reply:
I've intentionally separated it since it changes the ZAPI protocol & we know there are at least some external users (e.g. gobgp)